### ISSUE TITLE
SW page shows maintenance error.

### ISSUE THREAD

**Issue by pandabear41** [samyun/southwest-price-drop-bot#59]
I got the proxy configured with the heroku app. 
When using the application it logs SW is under maintenance. 
When doing ssh tunnel to the server that hosts the squid proxy the exact url that the app sends works.
I wonder if they are detecting and blocking puppet? Or did I not configure it correctly?
`<h2>Thanks For Stopping By!</h2><p>Getting you to where you want to go matters to us. However, this portion of the Southwest Airlines website is undergoing maintenance and is currently unavailable.&nbsp; We are working to make it available again soon.&nbsp; We apologize for any inconvenience this may cause, but please try again later.</p> `

**Comment by samyun** [comment]
This is new to me. Are you always getting this? Or does it come and go?

**Comment by pandabear41** [comment]
@samyun Yes, Every time I try to use this tool I get Maintenance errors for every request. Even tried a new server to host a proxy and a different server in a different subnet and geo.

**Comment by evliu** [comment]
@pandabear41 try now. I added a more-realistic userAgent
